NUXT/plugins/languages/japanese.js

```diff
@@ -66,21 +66,21 @@ module.exports = {
         dark: "ダーク",
         black: "ブラック",
         darkmode: "ダークモード",
-        darkmodetagline: "Bravo Six, Going Dark.(ブラボー6 通信を終了する)",
+        darkmodetagline: "こちらブラボー・シックス、闇の中へ",

```

Comment:
Does this translate well? I'm not well-versed enough in Japanese meme culture to know whether Japanese-speaking people will understand the meme.

Reply:
Previous translations

> ```ブラボー6 通信を終了する``` : Bravo 6 terminate the communication.

This subtitle appears at 00:41-00:44 of this video : [CallofDutyJP](https://twitter.com/CallofDutyJP/status/1134171406277763072) (Twitter)

This is faithful, but general people don't understand the meaning. We need video as well as text. The sentence "terminate the communication" will confuse some people.

----
New Translation

> ```こちらブラボー・シックス、闇の中へ``` : I'm Bravo six, Into the Dark. 

It's an operational style of plunging into darkness. It can be understood by general people without video. Japanese for Bravo Six : ```"ブラボー・シックス"``` : Searching for this You can reach the Japanese version at https://en.wikipedia.org/wiki/Captain_Price.

If you want to keep the original English text and make it easier to search:

```こちらブラボー・シックス、闇の中へ (Bravo Six, Going Dark)```

Reply:
Sounds good then 👍